compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -72,6 +72,21 @@ static void populateConvertGPUToAMDGPUPatterns(RewritePatternSet &patterns) {
 
 } // namespace
 
+// Function to check valid data types on the ROCm backend.
+static LogicalResult validateDataTypes(Operation *op) {
+  if (llvm::any_of(op->getOperandTypes(),
+                   llvm::IsaPred<Float8E4M3FNType, Float8E5M2Type>) ||
+      llvm::any_of(op->getResultTypes(),
```

Comment:
I think you can combine these two with `llvm::concat<Type>(op.getOperandTypes(), op.getResultTypes())`

Reply:
that's neat, thanks for the pointer :)